bindings/protocols/coap/coap.ttl

```diff
@@ -0,0 +1,89 @@
+@prefix cov: <https://www.w3.org/2019/wot/coap#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@base <https://www.w3.org/2019/wot/coap> .
+
+<> a owl:Ontology ;
+   dcterms:title "CoAP Vocabulary for the Web of Things"@en ;
+   rdfs:label "CoAP Vocabulary for the Web of Things"@en ; # TODO: Refactor
+   dcterms:creator [ a dcterms:Agent ,
+                       schema:Person ;
+                     schema:name "Klaus Hartke" ;
+                     schema:affiliation [ a schema:Organization ;
+                                          schema:name "Siemens AG" ;
+                                          schema:url "https://www.siemens.com/"
+                                        ]
+                   ] ,
+                   [ a dcterms:Agent ,
+                       schema:Person ;
+                     schema:name "Jan Romann" ;
+                     schema:affiliation [ a schema:Organization ;
+                                          schema:name "Invited Expert" ;
+                                          schema:url "https://github.com/JKRhb/"
+                                        ]
+                   ] ;
+   vann:preferredNamespacePrefix "cov" ;
+   vann:preferredNamespaceUri cov: ;
```

Comment:
```suggestion
   vann:preferredNamespaceUri "https://www.w3.org/2019/wot/coap#" ;
```

The value of `vann:preferredNamespaceUri` should probably be a string. [VANN doesn't say so explicitly](https://vocab.org/vann/) but it's [how VANN uses it itself](https://vocab.org/vann/vann-vocab-20100607.rdf). It also makes more sense than an IRI since `https://www.w3.org/2019/wot/coap#` does not refer to a node in the RDF graph here but is intended to be prepended to names with the `cov:` prefix.